site_ifoundadog/templates/401.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{% block title %}404{% endblock %}
```

Comment:
could you change the number from "404" to "401"?

Reply:
Thanks for looking at that Chris, I completed these. Please let me know if there's anything else!